src/Pact/Analyze/Types/Eval.hs

```diff
@@ -140,6 +140,9 @@ newtype Constraints
 instance Show Constraints where
   show _ = "<symbolic>"
 
+instance Semigroup Constraints where
+  (Constraints act1) <> (Constraints act2) = Constraints $ act1 *> act2
```

Comment:
If this is indeed required, please reuse code from `mappend` or vice versa.

Reply:
Sure, I will update this.

Reply:
Updated.